orbit/pkg/useraction/mdm_migration_darwin.go

```diff
@@ -102,7 +102,7 @@ type baseDialog struct {
 }
 
 func newBaseDialog(path string) *baseDialog {
-	return &baseDialog{path: path, interruptCh: make(chan struct{})}
+	return &baseDialog{path: path, interruptCh: make(chan struct{}, 1)}
```

Comment:
Why is this change needed?

Reply:
This took a while to track down, but essentially there's nothing listening on this channel unless the migrator is opened. The problem with that is that the `mdmMigrator.Exit()` method we call in `onExit` belongs to baseDialog, which sends an empty struct through the channel. We would just get stuck indefinitely because when nothing is listening to a chennel, sending blocks

Reply:
Ideally we should rewrite the mdm migrator using the new `swiftDialog` module and scrap what's going on in there entirely, but until then this works